CONTRIBUTING.md

```diff
@@ -110,40 +107,39 @@ you might find out that you don't need to create one.
 
 When you are creating an enhancement suggestion, please 
 [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). 
-Fill in the [issue_template.md](.github/issue_template.md), including the steps that you imagine you 
-would take if the feature you're requesting existed.
 
 #### Before Submitting An Enhancement Suggestion
 
 * **Check the [Debugging documentation].** You might be able to find the cause of the problem and fix things yourself. 
-* **Perform a [cursory search of project issues](https://github.com/search?q=+is%3Aissue+repo%3APegasysEng/Pantheon)** 
+* **Perform a [cursory search of project issues](https://pegasys1.atlassian.net/secure/Dashboard.jspa)** 
 to see if the problem has already been reported. If it has **and the issue is still open**, add a comment 
 to the existing issue instead of opening a new one.
 
 #### How Do I Submit A (Good) Enhancement Suggestion?
 
-Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/).
-Issues should provide the following information by filling in the [issue_template.md](.github/issue_template.md) 
-and providing the following information:
+Enhancement suggestions are tracked as [Jira issues](https://pegasys1.atlassian.net/secure/Dashboard.jspa).
```

Comment:
This will always load their last looked at dashboard. Do we want to specifically send them to the system dashboard (https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10000)?

Reply:
Yes - fixed